src/@chakra-ui/gatsby-plugin/components/Button.ts

```diff
@@ -26,6 +26,28 @@ const commonOutline = {
   },
 }
 
+const quizButton = {
```

Comment:
For me, this is a blocker. Calling @nloureiro or @konopkja to jump into the discussion.

Have we decided on creating a new button style just for this case? Can't we reuse a button style from the already existing styles? https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System

The problem I see with this new style is that we are falling into the problems we had before. We are creating individual styles for each implementation we are doing and we end up having a bunch of different button styles across the site. IMO we should try to use the styles that the Design System tells us.

Regardless of that, if we decide to implement this new style, I would:
1. Update the design system showing this new style
2. Change the name of this variant to something that is abstracted from the implementation. This maybe is our `ghost` variant
3. Do not use the suffix `Button` on the name as it is redundant